### DISCUSSION TITLE
turboacc出现问题

### DISCUSSION THREAD

**Discussion by ghost** [Boos4721/openwrt#160] [Bugs/问题反馈]
之前ax6反复重启的问题经测试已修复，感谢...
但turboacc中的软件流量分载始终是未运行，编译选的是sfe，不知是哪里的问题，求解

**Comment by Boos4721** [comment]
https://github.com/Boos4721/luci/commit/0f2b1d8feaccc9d299b59edb067127661f2e25bd

**Reply by ghost** [reply]
Oh, I see...thx
